pages/index.js

```diff
@@ -1,69 +1,479 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import Head from "next/head";
+import Image from "next/image";
+import styles from "../styles/Home.module.css";
+import styled from "styled-components";
+import Button from "../components/Button";
+import { useRef } from "react";
+import SVG1 from '../public/svg/LandingPage/svg1';
+import SVG2 from '../public/svg/LandingPage/svg2';
+import SVG3 from '../public/svg/LandingPage/svg3';
+import SVG4 from '../public/svg/LandingPage/svg4';
+import SVG5 from '../public/svg/LandingPage/svg5';
+import SVG6 from '../public/svg/LandingPage/svg6';
+import SVG7 from '../public/svg/LandingPage/svg7';
+import SVG8 from '../public/svg/LandingPage/svg8';
+import SVG9 from '../public/svg/LandingPage/svg9';
```

Comment:
Can you name these files and components as more descriptively.

Reply:
Will do this